### ISSUE TITLE
GitVersion.Portable throws when trying to init a new version number

### ISSUE THREAD

**Issue by asbjornu** [GitTools/GitVersion#514]
I installed version 3 beta 4 of [GitVersion.Portable](https://chocolatey.org/packages/GitVersion.Portable) to do `gitversion init` in a project where I wanted to control the minting of the next version number. Here's the command line output:
```
C:\...> gitversion init
GitVersion init will guide you through setting GitVersion up to work for you

Which would you like to change?

0) Save changes and exit
1) Exit without saving

2) Run getting started wizard

3) Set next version number
4) Branch specific configuration
5) Branch Increment mode (per commit/after tag) (Current: ContinuousDelivery)
6) Assembly versioning scheme (Current: MajorMinorPatch)
7) Setup build scripts

> 3

What would you like to set the next version to (enter nothing to cancel)?

> 1.1
ERROR [07/17/15 11:52:51:27] An unexpected error occurred:
System.ArgumentException: Unrecognised format 't'
Parameter name: format
   at GitVersion.SemanticVersion.ToString(String format, IFormatProvider formatProvider) in c:\projects\gitversion\GitVersionCore\SemanticVersion.cs:line 273
   at GitVersion.Configuration.Init.SetNextVersion.HandleResult(String result, Queue`1 steps, Config config, String workingDirectory, IFileSystem fileSystem) in c:\projects\gitversion\GitVersionCore\Configuration\Init\SetNextVersion.cs:line 21
   at GitVersion.Configuration.Init.Wizard.ConfigInitWizardStep.Apply(Queue`1 steps, Config config, String workingDirectory, IFileSystem fileSystem) in c:\projects\gitversion\GitVersionCore\Configuration\Init\Wizard\ConfigInitWizardStep.cs:line 31
   at GitVersion.Configuration.Init.Wizard.ConfigInitWizard.Run(Config config, String workingDirectory, IFileSystem fileSystem) in c:\projects\gitversion\GitVersionCore\Configuration\Init\Wizard\ConfigInitWizard.cs:line 18
   at GitVersion.ConfigurationProvider.Init(String workingDirectory, IFileSystem fileSystem) in c:\projects\gitversion\GitVersionCore\Configuration\ConfigurationProvider.cs:line 45
   at GitVersion.Program.VerifyArgumentsAndRun() in c:\projects\gitversion\GitVersionExe\Program.cs:line 74
ERROR [07/17/15 11:52:51:27] An unexpected error occurred:
System.ArgumentException: Unrecognised format 't'
Parameter name: format
   at GitVersion.SemanticVersion.ToString(String format, IFormatProvider formatProvider) in c:\projects\gitversion\GitVersionCore\SemanticVersion.cs:line 273
   at GitVersion.Configuration.Init.SetNextVersion.HandleResult(String result, Queue`1 steps, Config config, String workingDirectory, IFileSystem fileSystem) in c:\projects\gitversion\GitVersionCore\Configuration\Init\SetNextVersion.cs:line 21
   at GitVersion.Configuration.Init.Wizard.ConfigInitWizardStep.Apply(Queue`1 steps, Config config, String workingDirectory, IFileSystem fileSystem) in c:\projects\gitversion\GitVersionCore\Configuration\Init\Wizard\ConfigInitWizardStep.cs:line 31
   at GitVersion.Configuration.Init.Wizard.ConfigInitWizard.Run(Config config, String workingDirectory, IFileSystem fileSystem) in c:\projects\gitversion\GitVersionCore\Configuration\Init\Wizard\ConfigInitWizard.cs:line 18
   at GitVersion.ConfigurationProvider.Init(String workingDirectory, IFileSystem fileSystem) in c:\projects\gitversion\GitVersionCore\Configuration\ConfigurationProvider.cs:line 45
   at GitVersion.Program.VerifyArgumentsAndRun() in c:\projects\gitversion\GitVersionExe\Program.cs:line 74
```
I tried running it again and selecting `0) Save changes and exit` to have it create `GitVersionConfig.yaml` before incrementing the version number, but selecting `3) Set next version number` still fails with the above exception. Any ideas why this is happening and what I can do to fix it?

**Comment by JakeGinnivan** [comment]
Clearly I have not tested the code I wrote... Will look at it tomorrow and fix for stable. Sorry bout that

Sent from my Windows Phone
________________________________

**Comment by asbjornu** [comment]
Coupled with the suddenly vanished documentation, it was a bit interesting to figure out how the version was specified in the config file, but I found the file in a couple of your other projects, so I managed to get around the problem. No rush to fix it as long as this is squashed before stable.

But something should probably be done with regards to (automatically) testing the command line stuff. :smile:
